pdf/pdfcanvas.cpp

```diff
@@ -34,12 +34,13 @@ struct PDFPage {
     PDFPage()
         : index{-1}
         , requested{ false }
+        , rectPlaceholder{ false }
         , texture{ nullptr }
     { }
 
     int index;
     QRectF rect;
-    bool requested;
+    bool requested, rectPlaceholder;
```

Comment:
As personal preference, I'd keep member variable declarations separate to ease reading and follow existing convention. Doesn't matter much, though.